python_modules/dagster/dagster/cli/dynamic_loader.py

```diff
@@ -208,25 +208,47 @@ def create_pipeline_loading_mode_data(info):
         raise InvalidPipelineLoadingComboError()
 
 
-DynamicObject = namedtuple('DynamicObject', 'module fn module_name fn_name object')
+class DynamicObject:
+    def __init__(self, module, module_name, fn_name):
+        self.module = module
+        self.module_name = module_name
+        self.fn_name = fn_name
+        self.object = None
+        self.coerce_to_repo = False
+        self.loaded = False
+
+    def eval(self):
```

Comment:
i think we should use a different word here. eval generally has a definition of "take a string and val it as code". Let's figure out some names in person.

Reply:
good call—technically, I think this is the step where the python interpreter actually reads the contents of the module and finds that the syntax is invalid, etc. but I'm no Python expert and this certainly isn't "eval" in the JS sense. Will rename this to `load` for now since that's more directly what it's doing.